web/params.go

```diff
@@ -363,3 +357,25 @@ func ParamsFromRequest(r *http.Request) *Params {
 
 	return params
 }
+
+// getPerPageFromQuery returns the PerPage value from the given query.
+// This function should be removed and the support for `pageSize`
+// should be dropped after v1.46 of the mobile app is no longer supported
```

Comment:
I'd suggest to either add a `TODO` or the ticket number so that we make this easily discoverable later on.